packages/now-static-build/src/index.ts

```diff
@@ -327,6 +328,22 @@ export async function build({
       debug(
         `Detected ${framework.name} framework. Optimizing your deployment...`
       );
+
+      if (process.env.VERCEL_ANALYTICS_ID) {
+        const frameworkDirectory = path.join(
+          workPath,
+          path.dirname(entrypoint)
+        );
```

Comment:
Should we also consider monorepos here where the package.json is at the repo root instead of the workpath?


```js
  const packagejson = await walkParentDirs({
    base: repoRoot,
    start: frameworkDirectory,
    filename: 'package.json',
  });
```

Reply:
Good call. I'm not as familiar with this, could you possibly send the PR for this and add a test case?